models/experimental/functional_bloom/README.md

```diff
@@ -0,0 +1,21 @@
+# ttnn_functional_bloom Demo
+
+## How to Run
+
+Use `pytest --disable-warnings models/experimental/functional_bloom/demo/demo.py::test_demo[models/experimental/functional_bloom/demo/input_data.json-models.experimental.functional_bloom.tt.ttnn_functional_bloom]` to run the ttnn functional bloom demo.
+
+Use `pytest --disable-warnings models/experimental/functional_bloom/demo/demo.py::test_demo[models/experimental/functional_bloom/demo/input_data.json-models.experimental.functional_bloom.tt.ttnn_optimized_functional_bloom]` to run the ttnn optimized functional bloom demo.
+
+If you wish to run the demo with a different input use `pytest --disable-warnings models/experimental/functional_bloom/demo/demo.py::test_demo[address_to_your_json_file.json-path_to_functional_bloom]`.This file is expected to have exactly 8 inputs.
+
+Our second demo is designed to run HellaSwag dataset, run this with `pytest --disable-warnings models/experimental/functional_bloom/demo/demo.py::test_demo_hellaswag[4-models.experimental.functional_bloom.tt.ttnn_functional_bloom]` for ttnn functional bloom demo and run this `pytest --disable-warnings models/experimental/functional_bloom/demo/demo.py::test_demo_hellaswag[4-models.experimental.functional_bloom.tt.ttnn_optimized_functional_bloom]` for ttnn optimized functional bloom.
```

Comment:
Can you explain what is number 4 in parametrers?
And if I change it the test fails immediately.
```
ERROR: not found: /home/boris/tt-metal/models/experimental/functional_bloom/demo/demo.py::test_demo_hellaswag[2-models.experimental.functional_bloom.tt.ttnn_optimized_functional_bloom]
(no name '/home/boris/tt-metal/models/experimental/functional_bloom/demo/demo.py::test_demo_hellaswag[2-models.experimental.functional_bloom.tt.ttnn_optimized_functional_bloom]' in any of [<Module models/experimental/functional_bloom/demo/demo.py>])
```